tests/opentracer/utils.py

```diff
@@ -1,18 +1,39 @@
+from ddtrace.opentracer import Tracer, set_global_tracer
+
 from tests.test_tracer import get_dummy_tracer
 
 
-def opentracer_init(service_name="ot_svc", dummy_tracer=None):
+def get_dummy_ot_tracer(
```

Comment:
because we're using `pytest`, why not writing a fixture? We should use fixtures instead of plain functions, that's one of the cool things of `pytest`.